styles/github/catppuccin.user.css

```diff
@@ -501,6 +501,23 @@
     --shadow-floating-legacy: 0px 6px 12px -3px @crust, 0px 6px 18px 0px @crust;
     --outline-focus: @blue solid 2px;
 
+    /* Customer Agreements */
+    [data-color-mode="light"][data-light-theme="light"] .enterprise-customer-agreement, [data-color-mode="dark"][data-dark-theme="light"] .enterprise-customer-agreement {
+      color: @accent-color !important;
+    }
+
+    .enterprise-customer-agreement:hover .octicon {
+      color: @accent-color !important;
+    }
+
+    .enterprise-customer-agreement {
+      color: @accent-color !important;
+    }
+
+    .enterprise-customer-agreement h3:hover {
+      text-decoration: underline !important;
+    }
```

Comment:
Please nest this CSS, and you don't need the `[data-color-mode="___"]` stuff.

Reply:
Done! Commit ce9c3d1, ty for the quick reply.